lib/common/client/middleware/response/mhv_xml_html_errors.rb

```diff
@@ -5,17 +5,33 @@ module Middleware
       module Response
         class MhvXmlHtmlErrors < Faraday::Response::Middleware
           include SentryLogging
-          attr_reader :status
+          attr_reader :status, :breakers_service
+
+          def initialize(app, options = {})
+            # set the error prefix to something like 'RX' or 'SM'
+            @breakers_service = options[:breakers_service]
+            # raise ArgumentError.new('no BreakersService provided') unless @breakers_service.is_a?(Breakers::Service)
+            super(app)
+          end
 
           def on_complete(env)
             return if env.success?
             return unless env.response_headers['content-type'] =~ /\b(xml|html)/
             @status = env.status.to_i
             @body = env.body.delete('%') # strip percentages from html because Sentry uses it for interpolation
+            @header = env.response_headers.to_s.delete('%')
 
-            extra_context = { original_status: @status, original_body: @body }
+            # Logging the header as well, because it can include a Retry-After in the header if known
+            # This might be useful for improved handling with breakers
+            extra_context = { original_status: @status, original_body: @body, original_header: @header }
+            # eventually we won't need to log_message_to_sentry for 503, but lets do it for now
             log_message_to_sentry('Could not parse XML/HTML response from MHV', :warn, extra_context)
-            raise Common::Exceptions::BackendServiceException.new('VA900', response_values, @status, @body)
+            if status == 503
+              breakers_service.begin_forced_outage!
+              raise Breakers::OutageException.new(breakers_service.latest_outage, breakers_service)
```

Comment:
It looks like what you can do is:

```Breakers::Outage.create(breakers_service)```

That'll create a new outage and seems to not violate the encapsulation. It seems like the combination of doing that and raising the exception ought to be what you want. Creating the outage will notify statsd that it's down, and raising the exception will notify statsd that there was an exception.